### PR TITLE
Api

### PR DESCRIPTION
**Pull request by mubstimor**
A user can edit their orders
Included more tests.
Duplicate orders cannot be submitted.
Testing is done with more data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+
+* The Admin user should be able to do the following:
+See a list of orders
+Accept and decline orders
+Mark orders as completed
+
+PUT /orders/<orderId>
+
+* A user should be able to order for food
+POST /orders
```
Comment (jeanjoe) [inline README.md]: I have tested your post endpoint and I get `"error": "Bad Request,     some parameters are either missing or invalid"` message and I do not know which data to provide, can you please add them to the README or return them in the response.

**Review by jeanjoe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mubstimor) [reply README.md]: sure thing, thanks. I'll update the readme

Comment (mubstimor) [reply README.md]: For now, you can view the format of parameters in my test classes.

**Comment by jeanjoe** [comment]
Nice workflow, Just update your README.md

**Comment by mubstimor** [comment]
Have updated it now, thank you so much for that feedback.

**Pull request closed without merging**